__tests__/app.test.js

```diff
@@ -125,14 +129,73 @@ describe("/api/articles", () => {
                 expect(articles[0]).toHaveProperty("comment_count", '2');
             });
         });
-        test("Returns 'Status: 404' if invalid path provided to articles, e.g. misspelt path", () => {
-            return request(app).get('/api/articlees').expect(404);
+    });
+});
+
+describe("/api/articles/:article_id", () => {
+    describe("GET", () => {
+        test("Returns 'Status: 200' with single article object if valid article ID", () => {
+            return request(app).get('/api/articles/1').expect(200)
+            .then(({ body }) => {
+                const article = body.requestedArticle;
+                expect(typeof article).toBe("object");
+                expect(Array.isArray(article)).toBe(false);
+            });
+        });
+        test("Returns an article object containing the correct keys", () => {
+            return request(app).get('/api/articles/1').expect(200)
+            .then(({ body }) => {
+                const article = body.requestedArticle;
+                expect(article).toHaveProperty("author", expect.any(String));
+                expect(article).toHaveProperty("title", expect.any(String));
+                expect(article).toHaveProperty("article_id", expect.any(Number));
+                expect(article).toHaveProperty("body", expect.any(String));
+                expect(article).toHaveProperty("topic", expect.any(String));
+                expect(article).toHaveProperty("created_at", expect.any(String));
+                expect(article).toHaveProperty("votes", expect.any(Number));
+                expect(article).toHaveProperty("article_img_url", expect.any(String));
+            });
+        });
+        test("Returns the correct article object for article ID 1", () => {
```

Comment:
I would argue that you don't need this test as well as the top one, OR that you could just do expect article to equal (and then just the entire object)

Reply:
Thank you, done the latter :)